examples/chip-tool-darwin/templates/tests/tests.js

```diff
@@ -174,12 +175,21 @@ function getTests() {
     'Test_TC_MOD_1_1',
   ];
 
+  const MultipleFabrics = [
+    // These tests all need PairWithQRCode
```

Comment:
nit: Can this be made as a `TODO:`?

Reply:
I created an issue: https://github.com/project-chip/connectedhomeip/issues/17857

Reply:
Made all the comments TODO.